docs/topics/mta-cli-run-multiple-apps.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mta-cli-run-multiple-apps_{context}"]
+= Running the {ProductShortName} {CLINameTitle} against multiple applications and generating a single report (Developer Preview)
+
+You can now run {mta-cli} against multiple applications and generate a combined report. This can save you time and give you a better idea of how to prepare a set of applications for migration.
+
+This feature is currently a Developer Preview feature.
+
+[IMPORTANT]
+====
+Support for providing a single report when analyzing multiple applications on the CLI is Developer Preview software only. Developer Preview software is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this software to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
+
+For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
```

Comment:
Should we create a snippet for this?

Reply:
Created, using an attribute for the software name.